openmc/data/endf.py

```diff
@@ -249,6 +267,50 @@ def get_tab2_record(file_obj):
 
     return params, Tabulated2D(breakpoints, interpolation)
 
+def get_intg_record(file_obj):
```

Comment:
Add another blank line before this function (PEP8: 2 lines between functions/classes at top level)